vispy/geometry/calculations.py

```diff
@@ -78,3 +80,57 @@ def _calculate_normals(rr, tris):
     size[size == 0] = 1.0  # prevent ugly divide-by-zero
     nn /= size[:, np.newaxis]
     return nn
+
+
+def resize(image, shape, kind='linear'):
+    """Resize an image
+
+    Parameters
+    ----------
+    image : ndarray
+        Array of shape (N, M, ...).
+    shape : tuple
+        2-element shape.
+    kind : str
+        Interpolation, either "linear" or "nearest".
+
+    Returns
+    -------
+    scaled_image : ndarray
+        New image, will have dtype np.float64.
+    """
```

Comment:
Is there a reason to add this? I see it can be useful, but it does not seem like we're using it?